content/lessons/chapter-10/updating-the-state-1.tsx

```diff
@@ -44,12 +44,17 @@ export default function UpdatingTheState1({ lang }) {
           <Text className="mt-4 text-lg md:text-xl">
             {t('chapter_ten.updating_the_state_one.paragraph_one')}
           </Text>
+          <hr className="my-4 h-[1px] w-full opacity-25" />
+
+          <Text className="mt-4 text-lg font-bold md:text-xl">
+            {t('chapter_ten.updating_the_state_one.heading_two')}
+          </Text>
           <ul className="ml-4 list-disc  font-nunito text-lg">
             <li>{t('chapter_ten.updating_the_state_one.refund_list_one')}</li>
             <li>{t('chapter_ten.updating_the_state_one.refund_list_two')}</li>
           </ul>
           <Text className="mt-4 text-lg font-bold md:text-xl">
-            {t('chapter_ten.updating_the_state_one.heading_two')}
+            {t('chapter_ten.updating_the_state_one.heading_three')}
```

Comment:
I think we can switch this back to heading two

Reply:
Oh you're right, something got borked in the merge. I will fix it!

Reply:
Don't worry about it, I'll fix it with a commit here.
``` suggestion
            {t('chapter_ten.updating_the_state_one.heading_two')}
```